packages/check/package.js

```diff
@@ -4,15 +4,17 @@ Package.describe({
 });
 
 Package.onUse(function (api) {
-  api.use(['underscore', 'ejson'], ['client', 'server']);
-  api.use('jquery', 'client');
+  api.use(['underscore', 'modules', 'ejson'], ['client', 'server']);
```

Comment:
I think you want to actually depend on `ecmascript` instead of `modules` here, especially if you are using `const` in the code.